assets/src/components/closeButton.tsx

```diff
@@ -7,6 +7,7 @@ interface Props {
 
 const CloseButton = ({ onClick }: Props) => (
   <button
+    data-testid="close-button"
```

Comment:
I considered `title`, and we definitely should have titles on things anyway, but since this component is used in so many places that opens up a bit of a can of worms. Also, design is currently going through and reworking a lot of basic Skate stuff with an eye towards better accessibility and desktop usability, so this is the sort of thing we're going to change in the near future in a less ad-hoc manner.